src/server/db_slice.cc

```diff
@@ -268,6 +270,11 @@ DbSlice::DbSlice(uint32_t index, bool caching_mode, EngineShard* owner)
   CreateDb(0);
   expire_base_[0] = expire_base_[1] = 0;
   soft_budget_limit_ = (0.3 * max_memory_limit / shard_set->size());
+
+  std::string keyspace_events = GetFlag(FLAGS_notify_keyspace_events);
+  LOG_IF(FATAL, !keyspace_events.empty() && keyspace_events != "Ex")
```

Comment:
the convention is to print `LOG(ERROR)` and `exit(1)` to avoid ugly stacktraces.